ide/diff/src/org/netbeans/modules/diff/builtin/visualizer/editable/EditableDiffView.java

```diff
@@ -1518,6 +1587,20 @@ public void run() {
                         support.firePropertyChange(DiffController.PROP_DIFFERENCES, null, null);
                         jEditorPane1.setCurrentDiff(diffs);
                         jEditorPane2.setCurrentDiff(diffs);
+
+                        lineEnding1 = detectLineEnding(source1);
+                        lineEnding2 = detectLineEnding(source2);
+
+                        boolean showLineEnding = lineEnding1 != null && lineEnding2 != null && !lineEnding1.equals(lineEnding2);
+
+                        if(showLineEnding) {
+                            lineEndingLabel1.setText("<html><strong style='background-color: " + String.format("#%02x%02x%02x", colorChanged.getRed(), colorChanged.getGreen(), colorChanged.getBlue()) + "'>" + lineEnding1 + "</strong></html>");
+                            lineEndingLabel2.setText("<html><strong style='background-color: " + String.format("#%02x%02x%02x", colorChanged.getRed(), colorChanged.getGreen(), colorChanged.getBlue()) + "'>" + lineEnding2 + "</strong></html>");
+                        }
+
+                        setSourceTitle(fileLabel1, title1);
+                        setSourceTitle(fileLabel2, title2);
```

Comment:
This is exactly doing, what was already done in the constructor. Thus this can be removed.